Gemfile

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
```

Comment:
noticed the quotes being changed, I have no strong opinion on this yet I do know that that is specifically set on our rubocop as well, would you either: 

- use the default that the original repo introduced
- use the default we're imposing on ourselves

thoughts?